src/components/text/_text-bits.scss

```diff
@@ -45,7 +45,8 @@ $includeHtml: false !default;
     }
 
     &--small {
-      font-size: rhythm(1);
+      @include uppercaseText(0);
+      font-size: rhythm(25 / 24);
```

Comment:
I think font sizes should be taken from: https://github.com/brainly/style-guide/blob/master/src/sass/_mixins.scss#L41

Reply:
👍

Reply:
i;m not sure how, because there is no variable in fontSizes which has 25px - i was asked to make it exactly that big

Reply:
![image](https://user-images.githubusercontent.com/39903772/41292138-00cd5834-6e52-11e8-82f0-95a47ad5e49b.png)